spec/libsass-todo-issues/issue_1720/error-dart-sass

```diff
@@ -1,4 +1,4 @@
-Error: expected "{".
+Error: expected "}".
   ,
 3 | }
   |  ^
```

Comment:
The others look good. This one I'm not sure about. Why is the closing curly brace expected here?

Reply:
This is a weird one. This is the input file:

https://github.com/sass/sass-spec/blob/4ccb052d391212d5f4b20d77473cbba96fd27813/spec/libsass-todo-issues/issue_1720/input.scss#L1-L3

The `}` on line 2 is commented out, so the `}` on line 3 closes the interpolation. The style rule, however, remains unclosed, so the parser looks for another `}`.

Reply:
Ah thanks. I looked right over the // when I glanced at the input.